lib/src/action_decoder.dart

```diff
@@ -0,0 +1,18 @@
+part of redux_remote_devtools;
+
+/// Interface for custom remote action decoding logic
+abstract class ActionDecoder {
+  const ActionDecoder();
+
+  // Converts a JSON payload from remote devtools to an action
+  dynamic decode(dynamic json);
+}
+
+/// An action decoder that simply passes through the JSON unmodified
+class NOPActionDecoder extends ActionDecoder {
```

Comment:
You may want to consider change the name to NopActionDecoder https://www.dartlang.org/guides/language/effective-dart/style#do-capitalize-acronyms-and-abbreviations-longer-than-two-letters-like-words